packages/framework/assets/styles/admin/todo.less

```diff
@@ -2,3 +2,61 @@
 * use this file for temporary css changes
 * content will be added at the end of css file
 */
+
+@expand-button-height: 22px;
+
+.cke-textarea {
+    display: none;
+}
+
+.input-wysiwyg-localized {
+    .cke-preview {
+        padding-left: 32px;
+    }
+}
+
+.cke-preview {
+
+    position: relative;
+    border: @input-border-size solid @color-border;
+    border-radius: @radius;
+
+    &-text {
+        width: 100%;
+        min-height: 80px;
+        padding: 1em;
+        overflow: visible;
+        cursor: text;
+
+        &-collapsed {
+            overflow: hidden;
+            max-height: 200px;
+        }
+    }
+
+    textarea {
+        display: none;
+    }
+
+    &-expand-button {
+        display: none;
+        text-align: center;
+        width: 100%;
+        height: @expand-button-height;
+        line-height: @expand-button-height;
+        background-color: @bg-info;
```

Comment:
```suggestion
        background-color: #ffda6b;
```

Reply:
This color is not flashing in the eyes but catches your look in order to tell you - Hey mate, if you wanna play with this CKEditor, click me! :)

Reply:
This button should not initialize ckeditor, but merely expand the preview. 
Is it different for you?

Reply:
I leave it to @Srnka392 to decide the best colors

Reply:
I used for this button `btn--primary` modification, which is the blue color version of the button